otterdog/blueprints/scorecard_integration.yml

```diff
@@ -0,0 +1,43 @@
+id: scorecard-integration
+name: Integrate Scorecard anaylsis
+description: |-
+  Integrate scorecard analysis into the self-service.
+type: scorecard_integration
+config:
+  repo_selector:
+    name_pattern:
+      - otterdog
```

Comment:
you need to select the repos for which you wanna have the integration.
You can use a fnmatch pattern or just the repo names.

We will also add support for custom properties, but its not there yet.

Reply:
Yeah, just spotted that as I committed it. Updated the branch - thanks